HotSpot/ParkingSearchViewController.m

```diff
@@ -70,8 +94,21 @@ - (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(N
 }
 
 - (NSInteger)tableView:(nonnull UITableView *)tableView numberOfRowsInSection:(NSInteger)section {
-    // I return 10 for now just to see if this method is working
-    return 10;
+    return self.listings.count;
+    
+//    // I return 10 for now just to see if this method is working
+//    return 10;
+}
+
+- (void)tableView:(UITableView *)tableView didSelectRowAtIndexPath:(NSIndexPath *)indexPath {
+    // perform segue
+    UIStoryboard *bookingStoryboard = [UIStoryboard storyboardWithName:@"Booking"
+                                                                bundle:nil];
+    UINavigationController *navigationController = [bookingStoryboard instantiateViewControllerWithIdentifier:@"detailsNavigationController"];
```

Comment:
hmmmm....I think the navigation controller should be on the home page itself, so you don't need one between the home page and the booking page. can you make that change?